src/items/itemsTable.component.tsx

```diff
@@ -589,12 +598,28 @@ export function ItemsTable(props: ItemTableProps) {
           variant="outlined"
           onClick={() => {
             setItemsDialogType('create');
+            setOpenDialogAsPrivilegedUser(false);
             table.setCreatingRow(true);
           }}
         >
           Add Item
         </Button>
 
+        {isPrivilegedUser && (
+          <Button
+            startIcon={<AddIcon />}
+            sx={{ mx: 0.5 }}
+            variant="outlined"
+            onClick={() => {
+              setItemsDialogType('create');
+              setOpenDialogAsPrivilegedUser(true);
+              table.setCreatingRow(true);
+            }}
+          >
+            Add Item as admin
```

Comment:
Looking at the capitalisation perhaps Admin should be capitalised too?
<img width="630" height="107" alt="image" src="https://github.com/user-attachments/assets/bda2cda0-10f9-47ee-945b-80c76f88468e" />